{"uuid":"65e1db2a-7c03-4c0e-99e5-4de16f0313b9","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["10a66f13-7e32-4479-ac16-adb0c13e2c60"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378148723,"stop":1673378148723}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378148739,"stop":1673378148739},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378149083,"stop":1673378149083},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378149217,"stop":1673378149219},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"07f2668e-3a02-461b-a6c4-7f1ea5efd0ab-attachment","type":"image/png"},{"name":"Myscreenshot","source":"be089219-34e9-45c4-a739-7c89b937168e-attachment"}],"parameters":[],"start":1673378149234,"stop":1673378149515}],"start":1673378148723,"stop":1673378149515}